lib/minheap.js

```diff
@@ -11,18 +11,31 @@ class MinHeap {
   }
 
   // This method adds a HeapNode instance to the heap
-  // Time Complexity: ?
-  // Space Complexity: ?
+  // Time Complexity: O(log n)
+  // Space Complexity: O(log n)
   add(key, value = key) {
```

Comment:
The method works, and you have the space complexity right because you're doing recursion.